### DISCUSSION TITLE
Logical Location returns not right.

### DISCUSSION THREAD

**Discussion by phamkhactu** [RapidAI/TableStructureRec#18] [Q&A]
I have same problem with [issue](https://github.com/AlibabaResearch/AdvancedLiterateMachinery/issues/66) when using ```LinelessTableRecognition```. Start row and end row are sometime right , not right. How can I solve this problem?

**Comment by SWHL** [comment]
This situation is generally due to the poor accuracy of the model itself. To solve this, it is usually necessary to fine tune the model.

**Reply by phamkhactu** [reply]
> This situation is generally due to the poor accuracy of the model itself. To solve this, it is usually necessary to fine tune the model.

Thanks for your reply. I have another question: "Have you converted structure-recognition_cycle-centernet to onnx". I want to test 2 models: LORE and Center to determine which is the most suitable model with my data to fintune.

Thank you.
